test/client/fixtures/babel-transform-test.js

```diff
@@ -1,14 +1,10 @@
 module('regression');
 
-asyncTest('should not throw an error after for...of loop transformation (GH-1231)', function () {
-    var iframe = document.createElement('iframe');
-    var div    = document.createElement('div');
+test('should not throw an error after for...of loop transformation (GH-1231)', function () {
+    return window.createTestIframe(window.getSameDomainPageUrl('../data/iframe/simple-iframe.html'))
```

Comment:
Methods should be added to the [.eslintrc](https://github.com/DevExpress/testcafe-hammerhead/blob/master/test/client/.eslintrc#L20) appropriately.